### DISCUSSION TITLE
How do I collect data from two forms at the same time with just one submit button?

### DISCUSSION THREAD

**Discussion by ReformedCola** [react-hook-form/react-hook-form#5798] [Q&A]
CSB: https://codesandbox.io/s/react-hook-form-basic-forked-to3co?file=/src/index.js

How do I collect the data from  `<FormA/>` and `<FormB/>` with the submit button outside of them?

**Comment by bluebill1049** [comment]
This is a tricky one, have you consider the validation to your end user? it's achievable tho.

you can probably use `handlSubmit(onSubmit)()` twice for both forms and have a local state to host both form values.

**Reply by ReformedCola** [reply]
I do have validations for both form data, and I want to validate(both forms at the same time) and collect the data once I hit the submit button. 

Using `handleSubmit(onSubmit){}` twice, is that mean to separate both forms and store their values within a state (like the state I created in my given example)? But how do I trigger both `handleSubmit(onSubmit){}` with one submit button?
